### PR TITLE
feat: ability to translate nested JSON

### LINKED ISSUE #1: Deeply nested JSON structures
Hi, firstly thanks for making this.

I tried to use your script and unfortunately it can't handle nested JSON objects like:

```json
{
  "Submit": "Odeslat",
  "signup": {
    "header": "Registrace nového salonu" << this is a problem
  },
}
```

Would be great to have support even for those cases.

### PR DESCRIPTION
**Pull request by MichalBryxi**
- A bit of refactoring to break the responsibilities
- Introduce `DEBUG=true|false` option to see what is happening
- Make it so that the script is able to translate a nested JSON
- Fixes #1

### CODE REVIEW

**Review by nandakumarrk — APPROVED** [review]

**Comment by MichalBryxi** [comment]
👋  Any problem with merging this?

**Review by kusaasira — APPROVED** [review]

**Review by CidQu — APPROVED** [review]

**Comment by Denwahome** [comment]
Please pull

**Comment by MichalBryxi** [comment]
@Denwahome "please pull"? Did you mean "please merge"? There is nothing in `EisenVault/master` that I could rebase onto.

**Comment by HiteshGit002** [comment]
> @Denwahome "please pull"? Did you mean "please merge"? There is nothing in `EisenVault/master` that I could rebase onto.

where to give api key

**Comment by MichalBryxi** [comment]
> > @Denwahome "please pull"? Did you mean "please merge"? There is nothing in `EisenVault/master` that I could rebase onto.
> 
> where to give api key

This implementation does not assume API key, it just queries the public interface. If you want to expand the functionality, PRs are welcome. Also it might be that it will just work by exploring how does the `@google-cloud/translate` library do this. Because this PR _only_ uses that library.

File: index.js [hunk 1]
```diff
@@ -1,55 +1,56 @@
+var fs = require("fs");
+const { constant } = require("lodash");
```
Comment (JonathonLangelUpkeep) [inline index.js]: This isn't used anywhere. Can this be deleted?

**Comment by MichalBryxi** [comment]
Closing for inactivity.

**Pull request closed without merging**